touchforms/backend/xformserver.py

```diff
@@ -259,10 +259,9 @@ def handle_request(content, server):
             result = xfsess.evaluate_xpath(content['xpath'])
             return {"output": result['output'], "status": result['status']}
         elif action == xformplayer.Actions.SYNC_USER_DB:
-            ensure_required_params(['username', 'hq_auth'], action, content)
-            trimmed_username = content['username'][:content['username'].index('.')]
-            result = touchcare.force_ota_restore(trimmed_username, auth=content['hq_auth'])
-            return result
+            ensure_required_params(['username', 'hq_auth', 'domain'], action, content)
+            trimmed_username = content['username'].split('@')[0]
```

Comment:
If its a web user this will be wrong, right? IE my username is still wpride@dimagi.com for the purposes of authing